app/models/season/phase_switcher.rb

```diff
@@ -1,9 +1,20 @@
+# frozen_string_literal: true
+
 class Season::PhaseSwitcher
 
   def self.season
     Season.current
   end
 
+  def self.destined(phase)
+    if phase.in?(whitelisted_phases)
```

Comment:
This is only personal preference, but rather then doing nothing on invalid input, I'd raise an error instead. I think this is more helpful in certain cases
```ruby
def self.destined(phase)
  raise ArgumentError.new("#{phase} is not a valid phase switch") unless whitelisted_phases.include?(phase)
  publis_send(phase)
end
```